08-transport.Rmd

```diff
@@ -170,12 +179,12 @@ aggregation of the origin-destination (OD) data to the level of residential zone
 ```{r}
 od = readRDS("extdata/bristol-od.rds")
 zones_attr = group_by(od, o) %>% 
-  summarise_if(is.numeric, sum) %>% 
+  summarize_if(is.numeric, sum) %>% 
   rename(geo_code = o)
 ```
 
 What just happened?
-After the `od` dataset was read-in, the chained operation:
+After the `od` dataset was read-in<!-- I think a short description of the od dataset might be helpful here -->, the chained operation:
```

Comment:
I know you say that a dataset description will follow in the next section. I think a description here would be rather helpful already here.